components/AriaTable/AriaTable.tsx

```diff
@@ -63,12 +91,36 @@ export const Th = forwardRef<ElementRef<typeof StyledTh>, ThProps>((props, ref)
 const StyledTd = styled('span', TableTd, {
   display: 'table-cell',
 });
-export const Td = forwardRef<
-  ElementRef<typeof StyledTd>,
-  Omit<ComponentProps<typeof StyledTd>, 'css'> &
-    VariantProps<typeof StyledTd> &
-    VariantProps<typeof TableTd> & { css?: CSS }
->((props, ref) => <StyledTd ref={ref} role="cell" {...props} />);
+export interface TdProps
+  extends Omit<ComponentProps<typeof StyledTd>, 'css'>,
+    VariantProps<typeof StyledTd>,
+    VariantProps<typeof TableTd> {
+  css?: CSS;
+  colSpan?: number;
+}
+export const Td = forwardRef<ElementRef<typeof StyledTd>, TdProps>(
+  ({ colSpan, css, ...props }, ref) => {
+    const colSpanCss = useMemo(
+      () =>
+        colSpan
+          ? {
+              width: `${Math.max(colSpan || 0, 0) * 100}%`,
```

Comment:
this seems to break with `table-layout: 'auto'`, investigating.

![image](https://user-images.githubusercontent.com/3367393/159724071-86415f29-d3c2-45a4-bf3d-52b9da181607.png)

Reply:
`float` hack does not work with `table-layout: 'auto'` because columns don't have the same width.

With absolute positionning, it is mandatory to keep a `visibility: 'hidden'` cell in the footer to ensure the footer does not collapse.